packaging/windows/registrykeys.wxs

```diff
@@ -25,7 +25,6 @@
         </RegistryKey>
 
         <Environment Id="E_PATH" Name="PATH" Value="[DOTNETHOME]bin" Part="last" Action="set" System="yes" />
-        <Environment Id="E_DOTNET_HOME" Name="DOTNET_HOME" Value="[DOTNETHOME]" Part="all" Action="set" System="yes" />
```

Comment:
@robmen Do you mean the `DOTNETHOME` id at https://github.com/dotnet/cli/blob/rel/1.0.0/packaging/windows/dotnet.wxs#L37?